### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Color Contrast: MAS1.4.3: Copy rights text in About Xamarin Profiler is not having color contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#415]
### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Navigate to "Xamarin Profiler" menu tab and invoke it.
3. Navigate to "About Xamarin Profiler" menu intem under "Xamarin Profiler" and invoke it.
4. This will invokes the "About Xamarin Profiler" Popup window.
5. Verify the color contrast ratio for "Copy right" text in About Xamarin Profiler

### Pre-Requisite:
1.Install Xamarin Profiler.
2.Launch Visual Studio.
3.Create a project and build the solution.
4.Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5.This will invokes "Xamarin Profiler" application.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 3.4:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshot 
![3 copy right text](https://user-images.githubusercontent.com/22956130/41590657-286418f0-73d5-11e8-9204-5fb2888adafc.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-MAC
Wipro
